src/plugins/plugin.colors.ts

```diff
@@ -85,20 +86,20 @@ export default {
 
   defaults: {
     enabled: true,
+    forceColorsPlugin: false
   },
 
-  beforeLayout(chart: Chart, _args, options: ColorsPluginOptions) {
+  beforeDatasetsUpdate(chart: Chart, _args, options: ColorsPluginOptions) {
     if (!options.enabled) {
       return;
     }
 
     const {
-      type,
       options: {elements},
       data: {datasets}
-    } = chart.config as ChartConfiguration;
+    } = chart.config;
 
-    if (containsColorsDefinitions(datasets) || elements && containsColorsDefinitions(elements)) {
+    if ((containsColorsDefinitions(datasets) || elements && containsColorsDefinitions(elements)) && !options.forceColorsPlugin) {
```

Comment:
```suggestion
    if (!options.forceColorsPlugin && (containsColorsDefinitions(datasets) || elements && containsColorsDefinitions(elements))) {
```

to do not run `containsColorsDefinitions` and `containsColorsDefinitions` if the option is `true`